### ISSUE TITLE
Brave Shield Blocks Videos from Loading and Pop-Ups Bypassing Shield

### ISSUE THREAD

**Issue by haflg19** [brave/browser-laptop#10218]
- Did you search for similar issues before submitting this one?
  Yes, see related issues at bottom, no issues for exact same problems though.

- Describe the issue you encountered:
  Some things that should not be are (only partly? idk.. the player is still visible but nothing plays) blocked by Brave Shields. For example, the videos in the feed under a twitch stream, only sit on a black screen which is pictured below. Another problem spot is on [KissAnime](https://kissanime.ru). All the videos and players that I have looked at on this site, still are able to pop-out some site or something when you click Play on the video, like they are supposed to... for a non-ad-blocking user. However, this obviously is supposed to be blocked with Brave Shield. On top of this, it seems that some players are not able to, or not able to efficiently, load videos while Brave Shield is on. For example, [Kimi No Na Wa on KissAnime](http://kissanime.ru/Anime/Kimi-no-Na-wa/Movie-1080p?id=137753&s=default)  is one where everytime I click Play (only happens once per page load, so refreshing the page will cause pop-out to come again) the pop-out site that is set to appear, still comes and is not blocked by Brave Shield. Then after closing this, and returning to the page, if you click to Play the video again it will show a loading sign but this seems to show endlessly and never actually load the video, seeing as I was able to sit on a black screen with the loading symbol for 5 mins, but as soon as I turned of Brave Shield, the video loaded fine. When going to any of these pages with Brave Shield turned off, the videos load fine, and obviously the pop-outs on [KissAnime](https://kissanime.ru) still happen, unless you use another adblocker.

  TL;DR something with Brave Shield, is blocking the videos in the feed on Twitch streams (under a stream, in the channel posts if you scroll down), and on other sites, like [KissAnime](https://kissanime.ru), pop-outs are not blocked and videos will not load while Brave Shield is on, it seems.

- Platform (Win7, 8, 10? macOS? Linux distro?):
  Windows 10

- Brave Version (revision SHA):
  Brave     0.18.14
  rev         ad92d02
  Muon     4.3.6

- Steps to reproduce:
    1. (Fresh Install of) Brave v0.18.14
    2. Brave Shields Up
    3. Find Site With Some (Video Related) Content (Example Sites Above)
    4. ????
    5.  Playing Content Does Not Work as Intended
    
- Actual result:
  Videos are not loading/playing... Pop-out ads are showing.

- Expected result:
  Videos load and then play once Play is clicked, pop-out ads are blocked and do not show while Brave Shields are up.

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
  Yes

- Is this an issue in the currently released version?
  Yes

- Can this issue be consistently reproduced?
  Yes

- Extra QA steps:
  What is this ^^(QA Steps)??

- Screenshot if needed:
![glitch](https://user-images.githubusercontent.com/25330146/28806019-b28271a0-763b-11e7-8078-ee0f34e39712.png)

- Related Issues:
  https://github.com/brave/browser-laptop/issues/10211 (same/similar issue, different site)

**Comment by luixxiul** [comment]
Re KissAnime: it is technically an ad though it is a pop-up messageBox. Also currently we don't block the 1st party ad. And, unless KissAnime is a pirate site, it is not worth fixing.

Re twitch: if you allow 3rd party cookie on the site via the lion icon, the video is loaded.

**Comment by Brave-Matt** [comment]
+1 from Reddit: https://www.reddit.com/r/brave_browser/comments/948myw/brave_ad_blocker_not_blocking_ads_on_kissanime/
